recipe/meta.yaml

```
@@ -10,27 +10,30 @@ source:
   sha256: 8281600da180ec8ae35427a4ab4f7b82bfec1e3d1e52f80cb60ea82b9512501c
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3.7
-    - regex
+    - python
+    - regex >=2022.3.15
 
 test:
   imports:
     - parsimonious
   requires:
+    - pip
     - pytest
   source_files:
     - parsimonious/tests
   commands:
-    - pytest parsimonious/tests
+    - pip check
+    - pytest parsimonious/tests --ignore=parsimonious/tests/test_benchmarks.py
```

```suggestion
    - pytest --pyargs parsimonious -v -k "not test_benchmarks.py"
```
Because we can remove `source_files` (since the tests get installed with the package), we can use `pytest --pyargs` to tell pytest to look for tests inside the `parsimonious` installed package.

---

ok that's cool, I did not know that,
I think the `-k` will not skip the file entirely while `--ignore` will, so I would preserve it:

```suggestion
    - pytest --pyargs parsimonious -v --ignore=tests/test_benchmarks.py
```